addon/components/dropzone-widget/component.js

```diff
@@ -107,8 +108,8 @@ export default Ember.Component.extend({
         });
     },
     didUpdateAttrs() {
-        if (this.get('enable') && !this.get('attached')) {
-            this.set('attached', true);
+        if (this.get('enable')) {
+            Dropzone.forElement(`#${this.elementId}`).destroy();
```

Comment:
It would be cleaner to save a reference to the Dropzone as a property in loadDropzone() and use it here.